eoscostarica/eos-rate
# style(home) apply max width to rows in home page

### Max width #126

#

### Steps to test

1. Open up the app in desktop.
1. Go to home page.
1. Resize the browser to notice the max width applied.

### Checklist

![home](https://user-images.githubusercontent.com/8497609/50531833-fa96a800-0ad5-11e9-83b6-018eac4979e5.gif)
